test/include/partitioning_custom_part_check_generated_include_test.rb

```diff
@@ -0,0 +1,977 @@
+#!/usr/bin/env rspec
```

Comment:
@imobach Is the test called during build? I don't think so, as it's not mentioned in Makefile...